src/xmipp/libraries/reconstruction/volume_subtraction.cpp

```diff
@@ -137,6 +143,7 @@ class ProgVolumeSubtraction: public XmippProgram
     	fnVol2A=getParam("--saveV2");
     	if (fnVol2A=="")
     		fnVol2A="volume2_adjusted.mrc";
+    	dontE=checkParam("--dont_compute_E");
```

Comment:
preferably `skipEnergy`  / `computeEnergy` (and set default parameter accordingly)

Reply:
I agree, I just need to modify other protocol then (PR https://github.com/I2PC/scipion-em-xmipp/pull/412)